Template.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>net5</TargetFramework>
+		<PackageType>Template</PackageType>
+		<PackageVersion>1.0.0</PackageVersion>
+		<PackageId>GmodNET.Templates</PackageId>
+		<Title>GmodDotNet Templates</Title>
+		<Authors>GmodNET</Authors>
+		<Description>Project Templates for GmodDotNet module</Description>
+		<PackageTags>dotnet-new;templates</PackageTags>
+		<IncludeContentInPack>true</IncludeContentInPack>
+		<IncludeBuildOutput>false</IncludeBuildOutput>
+		<ContentTargetFolders>content</ContentTargetFolders>
+	</PropertyGroup>
```

Comment:
Add `<NoBuild>true</NoBuild>` property. This will disable build at all.

Reply:
https://docs.microsoft.com/ru-ru/nuget/reference/msbuild-targets#packing-using-a-nuspec for more info.

Reply:
resolved by https://github.com/GmodNET/Templates/pull/2/commits/5820dfeee45bead39c96e3adf9658f01ca7c7376